src/ai/backend/manager/api/scaling_group.py

```diff
@@ -50,10 +50,10 @@ async def query_wsproxy_version(
 ) -> str:
     async with params.db_ctx.begin_readonly() as conn:
         query = (sa.select([scaling_groups])
-                   .where((scaling_groups.c.name == scaling_group_name), ))
+                   .where((scaling_groups.c.name == scaling_group_name)))
         result = await conn.execute(query)
         matched_sgroup = result.first()
```

Comment:
Just a minor update for more recently adopted code style please.
```suggestion
    async with params.db_ctx.begin_readonly() as conn:
        query = (
            sa.select([scaling_groups])
            .where(scaling_groups.c.name == scaling_group_name)
        )
        result = await conn.execute(query)
        matched_sgroup = result.first()
```